src/arrayMethodSort.js

```diff
@@ -5,7 +5,37 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const standartCompare = function(a, b) {
+      if (String(a) < String(b)) {
+        return -1;
+      }
+
+      if (String(a) > String(b)) {
+        return 1;
+      }
+
+      if (String(a) === String(b)) {
+        return 0;
+      }
```

Comment:
let's use ternary operator for this `-1 : 1`

Reply:
Thanks, it's done.